Products/ZenUI3/browser/resources/js/zenoss/BackCompatPanel.js

```diff
@@ -107,7 +114,36 @@ Ext.define("Zenoss.IFramePanel", {
             this.load(Ext.urlAppend(url,
                     '_dc=' + new Date().getTime()));
         }
+        // hide iframe el to avoid flickering on injectCss;
+        // show it on after load;
+        this.iframeEl.hide();
         this.waitForLoad();
+    },
+    /*
+    * this fn inject zen-cse.css into iframe document to make styles same as in main page
+    * we should call it every time on after iframe load;
+    * */
+    injectCss: function() {
+        var iframe = this.getFrame(),
+            html = document.getElementsByTagName('html')[0],
+            css = document.querySelector('[name=zen_cse_css]'),
+            cssC,
+            iframeHtml = iframe.contentDocument.getElementsByTagName('html')[0],
+            cseClasses = [];
+
+        Ext.each(html.classList, function(key, item) {
+            if (typeof key === 'string' && key.startsWith('z-cse')) {
+                cseClasses.push(key);
+            }
+        });
+
+        if (css && iframeHtml) {
+            cssC = css.cloneNode();
+            // add or multiple classes using ES6 spread syntax
+            iframeHtml.classList.add(...cseClasses);
```

Comment:
After much discussion, the consensus is to not introduce ES6 syntax into the codebase because this code may be backported to versions that don't support ES6 syntax.